flux-refineck/src/constraint_gen.rs

```diff
@@ -388,6 +402,7 @@ impl<'a, 'tcx> InferCtxt<'a, 'tcx> {
                 rcx.check_pred(p2, self.tag);
                 self.subtyping(rcx, ty1, ty2);
             }
+            (TyKind::Closure(did1), TyKind::Closure(did2)) if did1 == did2 => {}
```

Comment:
hmmm this is interesting maybe we can delay the implementation of function subtyping and it's just a matter of figuring out the `ClosureSubsts` deal.

Reply:
hmmm . removing for now!